neurovault/apps/statmaps/tests/test_feat.py

```diff
@@ -87,13 +88,10 @@ def testFEAT_NIDM(self):
                 if detect_feat_directory(root):
                     print 'Found FEAT directory at {}.'.format(root)
                     info['found_feat'] = True
-                    try:
-                        fslnidm = FSLtoNIDMExporter(feat_dir=root, version="0.2.0")
-                        fslnidm.parse()
-                        export_dir = fslnidm.export()
-                        ttl_file = os.path.join(export_dir,'nidm.ttl')
-                    except:
-                        print("Unable to parse the FEAT directory: \n{0}.".format(get_traceback()))
+                    fslnidm = FSLtoNIDMExporter(feat_dir=root, version="1.2.0")
```

Comment:
Would you like to use 1.3.0 (instead of 1.2.0) which is the latest NIDM-Results release? 

(In the next release of `nidmfsl`, we will make the exporter use the latest version by default (https://github.com/incf-nidash/nidmresults-fsl/issues/78)).